src/frontend/components/UI/TextInputField/index.tsx

```diff
@@ -45,7 +45,7 @@ const TextInputField = ({
       }
     }
     return
-  }, [input])
+  }, [input, onChange])
```

Comment:
While you're at it, seems like this is also the case:
```suggestion
  }, [input.current, onChange])
```

Reply:
Huh, TIL